src/constants/actions.ts

```diff
@@ -0,0 +1,35 @@
+export const propose = "Propose";
```

Comment:
I added these definitions so we can compare the title strings without worrying about them being wrong after being changed in the future